svalinn.yaml

```diff
@@ -147,67 +147,114 @@ log:
-  # value is not used.  If the multiple is set below 1, it defaults to 1.
-  # (Optional) defaults to 1
-  waitTimeMult: 5
-
-  # pingInterval is the time to wait in between pinging the database.  The ping 
-  # is a part of the health check for svalinn.  If pinging fails, svalinn will 
-  # report as unhealthy. If the ping interval is set to 0, it defaults to 1s.
-  # (Optional) defaults to 1s
-  pingInterval: 1s
-
-  # connectTimeout is the timeout for connecting to the database.  If 
-  # establishing a connection takes longer than this, it will fail.  If 
-  # the connectTimeout is set to 0, it defaults to 10s.
-  # (Optional) defaults to 10s
-  connectTimeout: 1m
-
-  # opTimeout is the timeout for database calls after svalinn is connected.
-  # If the opTimeout is set to 0, it defaults to 10s.
-  # (Optional) defaults to 10s
+########################################
+#   ycql config
+########################################
+#  # hosts is and array of address and port used to connect to the cluster.
+  hosts:
+    - "db"
+#  # username is the username to use when connecting to the database.
+#  # (Optional)
+#  username: "cassandra"
+#
+#  # password is the password to use when connecting to the database.
+#  # (Optional)
+#  password: "cassandra"
+#
+#  # database is the name of the database being connected to.
+#  database: "devices"
+#  # opTimeout is the timeout for database calls after svalinn is connected.
+#  # If the opTimeout is set to 0, it defaults to 10s.
+#  # (Optional) defaults to 10s
   opTimeout: 100ms
 
-  # maxIdleConns are the maximum number of idle connections allowed at one time.
-  # If a value below 2 is chosen, it defaults to 2.
-  # (Optional) defaults to 2
-  #maxIdleConns: 2
+#  # SSLRootCert is the root cert to use when connecting to the database.
+#  # The SSLKey and SSLCert must also be provided in order to connect securely.
+#  # (Optional)
+#  #sslRootCert: "/etc/svalinn/ca.crt"
+#
+#  # SSLKey is the SSL key to use when connecting to the database.  The
+#  # SSLRootCert and SSLCert must also be provided in order to connect securely.
+#  # (Optional)
+#  #sslKey: "/etc/svalinn/node.0.0.0.0.key"
+#
+#  # SSLCert is the SSL cert to use when connecting to the database.  The SSLKey
+#  # and SSLRootCert must also be provided in order to connect securely.
+#  # (Optional)
+#  #sslCert: "/etc/svalinn/node.0.0.0.0.crt"
+#
+#  # If you want to verify the hostname and server cert (like a wildcard for cass cluster) then you should turn this on
+#  # This option is basically the inverse of InSecureSkipVerify
+#  # See InSecureSkipVerify in http://golang.org/pkg/crypto/tls/ for more info
+#  # (Optional) defaults to false
+#  #enableHostVerification: false
 
-  # maxOpenConns are the maximum number of open connections allowed at one time.
-  # If the value is 0, there is an unlimited number of connections.
-  # (Optional)I
-  #maxOpenConns: 0
+
+########################################
+#   postgres config
```

Comment:
If we're going to support both (like you configure which one to use), I think we can leave this.  Otherwise, I think we should delete it.